src/views/accounts/AccountChangeSecret/index.vue

```diff
@@ -38,6 +38,12 @@ export default {
             name: 'ChangeSecretRecord',
             hidden: () => !this.$hasPerm('accounts.view_changesecretrecord'),
             component: () => import('@/views/accounts/AccountChangeSecret/ExecutionDetail/AccountChangeSecretRecord.vue')
+          },
+          {
+            title: this.$t('ChangeSecretAccount'),
+            name: 'ChangeSecretAccount',
+            hidden: () => !this.$hasPerm('accounts.view_changesecretexecution'),
+            component: () => import('@/views/accounts/AccountChangeSecret/AccountList.vue')
           }
         ]
       }
```

Comment:
The code appears to have no obvious differences between 2021 and 2025. It seems valid according to the knowledge cutoff. Please let me know if you need additional help from now forward.